frontend/src/main/components/Sections/SectionsTable.js

```diff
@@ -19,6 +19,16 @@ function getFirstVal(values) {
 }
 
 export default function SectionsTable({ sections }) {
+  const navigate = useNavigate();
+
+  // Stryker disable next-line all : TODO try to make a good test for this
+  const infoCallback = (cell) => {
+    navigate(
+      `/coursedetails/${qyyToYyyyq(cell.row.values.quarter)}/${
+        cell.row.values["section.enrollCode"]
+      }`,
+    );
+  };
   // Stryker enable all
   // Stryker disable BooleanLiteral
```

Comment:
This is too broad.  Please remove this and let's see what Stryker mutation exceptions this is papering over.